packages/cli/tests/unit/rpc/navie/thread/index.spec.ts

```diff
@@ -53,7 +53,13 @@ type ThreadPrivate = Omit<Thread, 'log' | 'listeners' | 'logEvent'> & {
 describe('Thread', () => {
   let thread: ThreadPrivate;
   let conversationThread: ConversationThread;
-  let mockNavie: INavie;
+  let mockNavie: {
+    providerName: string;
+    setOption: jest.Mock;
+    ask: jest.Mock;
+    on: jest.Mock;
+    terminate: jest.Mock;
+  };
```

Comment:
Why the change? It erases type information.

Reply:
https://github.com/getappmap/appmap-js/pull/2278/commits/977ab2d9f26ace80334a8e3102cf3982e2c58487